components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -2,6 +2,19 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/**********************************************************************
+
+  THIS FILE IS OBSOLETE AND UNMAINTAINED.
+
+  The functionality in this file has moved to a proper Gradle plugin
+  in components/service/glean-gradle-plugin.
+
+  However, this file must remain on the master branch indefinitely so that
```

Comment:
Gah :(

Reply:
Can we print a deprecation warning, like the Android plugin (and many parts of Gradle itself) do?

Reply:
Good idea.